src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequests.java

```diff
@@ -214,6 +216,39 @@ public ITextComponent getShortDisplayString()
         }
     }
 
+    public static class SmeltAbleOreRequest extends AbstractRequest<SmeltableOre>
+    {
+
+        private static final ImmutableList<ItemStack> oreExamples = ImmutableList.of(new ItemStack(Blocks.GOLD_ORE), new ItemStack(Blocks.IRON_ORE));
```

Comment:
This is not correct. Since this is 1.12 we can Access all items. Like we do in other requests. Please update so that all smeltable ores are present in the list.

Reply:
kk done